ibis/backends/tests/test_json.py

```diff
@@ -62,6 +64,20 @@ def test_json_getitem_array(json_t):
     assert result == expected
 
 
+@pytest.mark.notimpl(["polars"])
+@pytest.mark.notyet(
+    ["sqlite", "mysql", "risingwave"],
```

Comment:
Seems like `risingwave` works, and `pyspark` is the way to mark the pyspark backend.

```suggestion
    ["sqlite", "mysql", "pyspark"],
```